salt/states/pkg.py

```diff
@@ -743,7 +743,15 @@ def _find_install_targets(
     warnings = []
     failed_verify = False
     for package_name, version_string in desired.items():
-        cver = cur_pkgs.get(package_name, [])
+
+        # FreeBSD pkg supports `openjdk` and `java/openjdk7` package names
+        origin = bool(re.search('/', package_name))
```

Comment:
You don't need a regex to check if a substring is in a string; it can just be `has_origin = '/' in package_name`. However we might want to use a regex in order to not match cases like `/package_name` and `package_name/`